components/list/list.js

```diff
@@ -64,6 +64,14 @@ class List extends LitElement {
 		`;
 	}
 
+	getListItemCount() {
+		return this._getItems().length;
+	}
+
+	getListItemPosition(item) {
```

Comment:
Does this return 0 for the first item?  Just wondering if this is index or position, and what's going to get announced.

Reply:
Oh I see above that +1 is added to it.  I think I would call this `getListItemIndex`, but not a huge deal.  Usually I think of "position" starting at 1, but maybe that's just me.

Reply:
Oh yeah this does return the index that's why we increment the value in `parent.getListItemPosition(this) + 1`

Reply:
Oh yeah good point! I can fix up the language there.

Reply:
Index right now, which means we should either change the name to `Index` or return the position.